Content.Server/Singularity/StartSingularityEngineCommand.cs

```diff
@@ -26,6 +28,22 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
             }
 
             var entityManager = IoCManager.Resolve<IEntityManager>();
+            // Dear code reviewers: I don't think there's a good way to do this,
+            //  and the command exists to give fast turnaround time on testing singulo,
+            //  which, given all the singulo bugs, seems necessary,
+            //  so it can't just be removed.
+            // What this code does: Constructs all PA components by forcing node changes into the completed state.
+            foreach (var comp in entityManager.ComponentManager.EntityQuery<ConstructionComponent>().ToArray())
+            {
+                // Check for valid graph
+                var graph = comp.GraphPrototype;
+                if (graph == null)
+                    continue;
+                // Construct PAs
+                if (graph.ID.StartsWith("particleAccelerator") && graph.Nodes.ContainsKey("completed"))
+                    await comp.ChangeNode("completed");
+            }
+            // Turn on emitters and collectors.
```

Comment:
If we do have to do this mess don't make it an admin command because the possibility of running it on a live server is a terrible idea.

Reply:
It's this or manually setting up the PA every time I test, so are you saying something like a mapping commmand, or?